### PR TITLE
homepage

### PR DESCRIPTION
**Pull request by ELI-LOGINS**
A detail farmers market

### CODE REVIEW

**Comment by gogurlbaby** [comment]
### Required Changes ♻️

Hi @ELI-LOGINS.

Good job so far!
There are some issues that you still need to work on to go to the next task but you are almost there!

159483175-2f61461d-b352-4cb4-98f8-b173e05ab5ff

### Highlights

✔️ Correct File Structure

### Required

- [ ] _Check the comments under the review._

### Optional Suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@gogurlbaby ] in your question so I can receive the notification.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

**Comment by gogurlbaby** [comment]
### Apply These Changes Effectively

### Comments

1. Use comments to describe what you are working on and also to differentiate sections.
Eg.  
 ` <!-- Navbar Section-->`
 `<!-- Hero Section-->`

### NavBar Section

2. Nice Navbar 👍.

Add a padding to the hover and some border-radius to make it more appealing. 

### Hero Section

3. Allow the background image to cover the section.

- The title and description should be visible as soon as a user opens the site. Center it at the left section instead of below.

- The title and description should be short, concise and catchy.

- Add a CTA (Call to Action) Button after the description to start make users interact immediately with the site.

### Features Section

4. Nice Features Sections.

- Add enough margin bottom to the hero section to distinguish the features section from the hero section. Same applies to mobile and tablet.

- Add a shade of color to the price color to make it more visible.

### Footer Section

5. Very important to add the footer section. Work on that.

### Misc

6. Remove the font awesome library. Use only the needed font and icons. No need to install the whole library. It slows down the performance of your app.

7. Organise your images into a folder called images.


### Mobile Section

8. Center the text (title and paragraph).
 

- Add enough margin bottom to the navbar to distinguish it well from the hero section. Same for desktop.

- Use grid instead of flex on the parent div. On desktop, it can be 3 items per section, 2 items for tablet and  1 item for mobile.

Overall your work looks good. Well done! 👍

**Review by gogurlbaby — CHANGES_REQUESTED** [review]
Kindly make the requested changes ASAP so we take it from there, All the best!

**Comment by ELI-LOGINS** [comment]
After doing the changes should i commit it to github?

**Comment by gogurlbaby** [comment]
> After doing the changes should i commit it to github?

Yes you always have to push to github. If you don't push your changes, how willI I see them?

File: index.html [hunk 1]
```diff
@@ -12,11 +12,13 @@
     <script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
 </head>
```
Comment (gogurlbaby) [inline index.html:13]: Add a favicon to your head.

File: index.html [hunk 2]
```diff
@@ -12,11 +12,13 @@
...
 <body>
+    <!-- navbar section -->
    <div class="container">
       <div class="heada">
```
Comment (gogurlbaby) [inline index.html:17]: check and correct spelling of class name.

File: index.html [hunk 3]
```diff
@@ -12,11 +12,13 @@
...
             <div class="header-container">
-              <h2>Farmers Market</h2>
+              <h2>Farmers <span>Market</span></h2>
             </div>
+            <!-- shopping cart favicon -->
             <div>
                 <a href="#"><i class="fa-solid fa-cart-shopping" id="icon"></i></a>
```
Comment (gogurlbaby) [inline index.html:23]: Add enough width to the side menu for mobile. Use a side menu for tablet too.

File: index.html [hunk 4]
```diff
@@ -52,10 +55,18 @@ <h2>Farmers Market</h2>
         <div class="image">
```
Comment (gogurlbaby) [inline index.html:55]: The background image should cover the hero section.

File: index.html [hunk 5]
```diff
@@ -52,10 +55,18 @@ <h2>Farmers Market</h2>
...
             <div class="bg-image">
                 <h1>Hilltop Farm</h1>
```
Comment (gogurlbaby) [inline index.html:57]: Reduce font-size of tilte for mobile and tablet.

File: index.html [hunk 6]
```diff
@@ -52,10 +55,18 @@ <h2>Farmers Market</h2>
...
-                <h3>Organic farm in Ghana</h3>
+                <h3>The best among all organic farms in Ghana</h3>
+                 <div class="blue-btn"> <!--Get Started button -->
+                    <a class="first-link" href="">
+                      Get Started
```
Comment (gogurlbaby) [inline index.html:61]: Use only one CTA button. Not necessary to use get started and learn more to confude your users.

If you wish to add another CTA button for a purpose, show both buttons then.

File: index.html [hunk 7]
```diff
@@ -52,10 +55,18 @@ <h2>Farmers Market</h2>
         <div class="image">
             <div class="bg-image">
                 <h1>Hilltop Farm</h1>
-                <h3>Organic farm in Ghana</h3>
+                <h3>The best among all organic farms in Ghana</h3>
```
Comment (gogurlbaby) [inline index.html:58]: Reduce font-size of description for mobile and tablet.

File: index.html [hunk 8]
```diff
@@ -52,10 +55,18 @@ <h2>Farmers Market</h2>
...
+                 <div class="blue-btn"> <!--Get Started button -->
+                    <a class="first-link" href="">
+                      Get Started
+                    </a>
+                    <a href="">
+                    Learn More
+                    </a>
+                  </div>
             </div>
         </div>
```
Comment (gogurlbaby) [inline index.html:68]: Add more margin-bottom to hero section to distinguish it from features section.

File: index.html [hunk 9]
```diff
@@ -215,6 +226,34 @@ <h3>Organic Lettuce</h3>
 
             </div>
         </div>
+
+        <hr>
+        <!-- footer section -->
+        <footer>
```
Comment (gogurlbaby) [inline index.html:232]: Search for footer samples online to get inspiration from. Improve on your footer section to make it more appealing.

**Comment by gogurlbaby** [comment]
### Apply the Below Suggestion Immediately on the Features Section.

Use grid instead of flex on the parent div. On desktop, it can be 3 items per section, 2 items for tablet and 1 item for mobile.

### Responsiveness

The best practice of a good developer is styling starting with mobile, then you use media queries to style for tablet and desktop. So inspect in your browser, then start styling for mobile before moving to desktop. So move all your styling for desktop into media queries. And start the original styling with mobile. It makes it very easy. Start and you won't regret it.

### New Branch

After making the changes, create a new branch called fix-home before adding and committing, then push to the new branch.

**Review by gogurlbaby — CHANGES_REQUESTED** [review]
After making the changes, create a new branch called fix-home before adding and committing, then push to the new branch.

File: index.html (hunk 3 above)
Comment (ELI-LOGINS) [reply index.html:23]: should i use a media query to determine the screen size?

**Pull request merged**